lib/constants.ts

```diff
@@ -107,6 +107,12 @@ export const ENABLE_GROUNDTRUTH = true;
 // The default name for the application
 export const APPLICATION_NAME = 'MLSpace';
 
+// Policy names attached to NOTEBOOK_ROLE_ARN that restricts instance types that a notebook
+// can use for each service
+export const ENDPOINT_CONFIG_INSTANCE_CONSTRAINT_POLICY_ARN = '';
```

Comment:
I thought we were hoping to deprecate constants.ts. Should we avoid adding new things here?

Reply:
We're migrating *some* things from constants.ts, but this entire file won't be done away with. 

The idea is that in the future this file will only be deployment-time configurations, and then things like ENABLE_TRANSLATE and ENABLE_GROUNDTRUTH will be handled by app config.

So anything dealing with roles/policies will need to remain in here.